config/accounts.go

```diff
@@ -21,11 +21,12 @@ const (
 )
 
 // Account represents a publisher account configuration
+// EventsEnabled is defined as pointer bool so that we can distinguish between nil and false.
 type Account struct {
```

Comment:
Can we please add a test case similar to `TestMigrateConfigDatabaseQueryParams` found inside `config/config_test.go`  to assert that this `Account` struct gets unmarshalled by Viper correctly?